integrations/common/rest/src/main/java/io/helidon/integrations/common/rest/RestApi.java

```diff
@@ -376,7 +360,9 @@ protected B me() {
          * This implementation builds the web client and sets up JSON factories.
          */
         protected void preBuild() {
-            webClientBuilder.addMediaSupport(JsonpSupport.create());
+            webClientBuilder.mediaContext(MediaContext.builder()
+                                                      .addMediaSupport(JsonpSupport.create())
```

Comment:
The `addMediaSupport` method was added recently, you can now do this:

```java
Http1Client.builder()
           .addMediaSupport(JsonpSupport.create())
           .build();
```

I.e. you can leave this method unchanged

Reply:
Updated it accordingly.